src/source/authentication.rst

```diff
@@ -40,6 +40,11 @@ OAuth 2.0 defines the following roles of users and applications:
   In this context *authorization server* is the server hosting authentication
   and token endpoints.
 
+In our Oauth2 flow we rely on the *tenant* as being the **Authentication Server**, the
```

Comment:
Do we want to call this 'tenant' or 'local market' ?

Reply:
how about *tenant* (local market) :D

Reply:
what's the difference? the local market plays a tenant role, that's how I see it (for us)